website/content/docs/security/acl/tokens/create/create-a-terminating-gateway-token.mdx

```diff
@@ -0,0 +1,354 @@
+  --header "X-Consul-Token: $CONSUL_HTTP_TOKEN" \
+  --data '{
+  "Name": "tgw-register",
+  "Description": "Terminating gateway policy",
+  "Rules": "service \"terminating-gateway\" {\n   policy = \"write\"\n}\nservice \"external-service-1\" {\n  policy = \"write\"\n}\nservice \"external-service-2\" {\n  policy = \"write\"\n}\nnode_prefix \"\" {\n  policy = \"read\"\n}\nagent_prefix \"\" {\n  policy = \"read\"\n}\n"
+}'
+```
+
+Refer to [ACL Policy HTTP API](/consul/api-docs/acl/policies) for additional information about using the API endpoint.
+
+</Tab>
+
+</Tabs>
+
+### Link the policy to a token
+
+After registering the policy into Consul, you can create and link tokens using the Consul command line or API endpoint. You can also enable Consul to dynamically create tokens from trusted external systems using an auth method.
+
+<Tabs>
+
+<Tab heading="CLI" group="CLI">
+
+Run the `consul acl token create` command and specify the policy name or ID to create a token linked to the policy. Refer to [Consul ACL Token Create](/consul/commands/acl/token/create) for details about the `consul acl token create` command.
+
+The following command creates the ACL token linked to the policy `tgw-register`.
+
+```shell-session
+$ consul acl token create \
+    -description "Terminating gateway token" \
+    -policy-name "tgw-register"
+```
+
+</Tab>
+
+<Tab heading="API" group="API">
+
+Send a PUT request to the `/acl/token` endpoint and specify the policy name or ID in the request to create an ACL token linked to the policy. Refer to [ACL Token HTTP API](/consul/api-docs/acl/tokens) for additional information about using the API endpoint.
+
+```shell-session
+$ curl --request PUT http://127.0.0.1:8500/v1/acl/token \
+  --header "X-Consul-Token: $CONSUL_HTTP_TOKEN" \
+  --data '{
+  "Policies": [
+    {
+      "Name": "tgw-register"
+    }
+  ]
+}'
+```
+
+</Tab>
+
+</Tabs>
+
+## Consul Enterprise
+
+To create a token for the terminating gateway, you must define a policy, register the policy with Consul, and link the policy to a token.
+
+### Define a policy
+
+You can send policy definitions as command line or API arguments or define them in an external HCL or JSON file. The following example policy is defined in a file. Refer to [ACL Rules](/consul/docs/security/acl/acl-rules) for details about all of the rules you can use in your policies.
+
+You can specify an admin partition and namespace when using Consul Enterprise. The following example policy allows a terminating gateway to register as a service named `terminating-gateway` in namespace `ns1` in partition `ptn1`.
+
+For this example, the terminating gateway forwards traffic for the following two services:
+
+* `external-service-1` in the `default` namespace
+* `external-service-2` in the `ns1` namespace
+
+The example policy contains permissions for resources in multiple namespaces. You must create ACL policies that grant permissions for multiple namespaces in the `default` namespace.
```

Comment:
```suggestion
The example policy contains permissions for resources in multiple namespaces. You must create ACL policies that grant permissions for multiple namespaces in the `default` namespace.
```
Should we repeat the information about setting `service:write` permissions for any additional services? Do we need to include that statement at all or is it reasonable to assume that practitioners would understand that they need to add the permission for each service in their policy?